plugins/Morpheus/stylesheets/uibase/_headerMessage.less

```diff
@@ -9,7 +9,7 @@
   border-radius: 4px;
 
   &.update_available:not(.isPiwikDemo) .title {
-    color: #CA8100;
+    color: #fbf7f1;
```

Comment:
That seems wrong. 
![image](https://user-images.githubusercontent.com/1579355/47880122-a9926c80-de22-11e8-83c8-ba0d80949b7a.png)

Reply:
good point, should be `#f57c00`. I'll fix it